airsonic-main/src/main/java/org/airsonic/player/controller/CoverArtController.java

```diff
@@ -684,6 +684,10 @@ public String toString() {
             this.height = height;
 
             int hash = key.hashCode();
+            // http://findbugs.sourceforge.net/bugDescriptions.html#RV_ABSOLUTE_VALUE_OF_HASHCODE
+            if (hash == Integer.MIN_VALUE) {
+              hash++;
+            }
             int rgb = COLORS[Math.abs(hash) % COLORS.length];
```

Comment:
This could probably just be `int rgb = COLORS[(key.hashCode() & Integer.MAX_VALUE) % COLORS.length];`